pegasus/sites.v3/hourofcode.com/public/events.haml

```diff
@@ -28,8 +28,9 @@ layout: wide_index
       .text-wrapper.flex-1
         %h1.no-margin-top
           =hoc_s("hoc_events.heading")
+        -# TODO - Update this to use .desc_sept for all languages once translation comes in
         %p.body-two
-          =hoc_s("hoc_events.desc")
+          =hoc_s(@language == 'en' ? "hoc_events.desc_sept" : "hoc_events.desc")
```

Comment:
This is showing the original `October` string for non-EN:
<img width="1283" alt="Screenshot 2024-09-19 at 2 15 51 PM" src="https://github.com/user-attachments/assets/9ced648a-2cde-4552-a721-c2ce9f634bca">

Reply:
If you just keep the old string and update the english translation then it won't break the translations in other languages and they will get updated over time

Reply:
Oh yes, you're right 🤦‍♀️